code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MitrokhinNikitaTest.java

```diff
@@ -0,0 +1,73 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static org.junit.Assert.*;
+
+public class MitrokhinNikitaTest {
+    @Test
+    public void canAddComplexNumbers() {
+        // Arrange
+        ComplexNumber z1 = new ComplexNumber(2, 0);
+        ComplexNumber z2 = new ComplexNumber(0, 1);
+
+        // Act
+        ComplexNumber z = z1.add(z2);
+
+        // Assert
+        assertEquals(new ComplexNumber(2, 1), z);
+    }
+
+    @Test
+    public void canConvertComplexNumberToString() {
+        // Arrange
+        ComplexNumber z = new ComplexNumber(-1, -1);
+
+        // Act
+        String strComplex = z.toString();
+        var expectedString = "-1.0 - 1.0i";
+
+        // Assert
+        assertEquals(expectedString, strComplex);
+    }
+
+    @Test
+    public void canCompareComplexNumbers() {
```

Comment:
I think the more appropriate name is "canCompareEqualComplexNumbers".
It speaks better about the goal of the test.

Reply:
Ok, fixed it.